### ISSUE TITLE
How to reduce the training time of sac?

### ISSUE THREAD

**Issue by jaried** [aai-institute/tianshou#650]
-  I have marked all applicable categories:
-  I have visited the [source website](https://github.com/thu-ml/tianshou/)
-  I have searched through the [issue tracker](https://github.com/thu-ml/tianshou/issues) for duplicates
-  I have mentioned version numbers, operating system and environment, where applicable:
  ```python
  import tianshou, gym, torch, numpy, sys
  print(tianshou.__version__, gym.__version__, torch.__version__, numpy.__version__, sys.version, sys.platform)
0.4.8 0.21.0 1.11.0+cu113 1.20.3 3.9.7 (default, Sep 16 2021, 16:59:28) [MSC v.1916 64 bit (AMD64)] win32
```
I now use sac to train once until `loss/alpha` is close to 0, which basically takes `20 hours`. If only **a small part of data** is trained on **the basis of existing weights**, how can I **reduce** the training time of sac? Done within `1-2 hours`.

**Comment by Trinkle23897** [comment]
Have you use the `resume training` part?
https://github.com/thu-ml/tianshou/blob/277138ca5b050518aacaaea367192f910fbe666d/examples/mujoco/mujoco_sac.py#L129-L132

**Comment by jaried** [comment]
Yes, I have ckpt loaded.

I can only change `update_per_step` now, but some people say that the update frequency is better to be slower, about 10-20 steps to update the best, but it will take longer.

```python
                checkpoint = torch.load(ckpt_file, map_location=args.device)
                policy.load_state_dict(checkpoint['model'])
                actor_optim.load_state_dict(checkpoint['actor_optim'])
                critic1_optim.load_state_dict(checkpoint['critic1_optim'])
                alpha_optim.load_state_dict(checkpoint['alpha_optim'])
```

**Comment by Trinkle23897** [comment]
Hmm... It is the property of SAC. What if you run a profile for your current program to see which part takes the most of the time? I guess it's still the environment.

**Comment by jaried** [comment]
I'll first see if changing the initial value of alpha solves the problem. #649

**Comment by jaried** [comment]
I ran `env` with random actions, and one episode takes `50 seconds`. It seems that env needs to be optimized.

I optimized the code, from `50 seconds` per episode to `30 seconds`.

```python
    episodes = 10
    for _ in tqdm.tqdm(range(episodes), total=episodes):
        status = env.reset()
        while True:
            state, reward, done, _ = env.step(env.action_space.sample())
            if done:
                break
```

**Comment by jaried** [comment]
I set alpha to the value it was before I interrupted training, and after reading the saved weights, I can resume training.
